src/ts/components/account/GeneratePhrase.tsx

```diff
@@ -116,10 +116,14 @@ class GeneratePhrase extends React.Component<IGeneratePhraseProps, IGeneratePhra
             <Grid columns='equal'>
               <Grid.Row>
                 <Grid.Column>
-                  <Button onClick={this.copyText}>
-                    <Icon name='copy' />
-                    {t('copyText')}
-                  </Button>
+                <Popup
+                  trigger={<Button><Icon name='copy' />{t('copyText')}</Button>}
```

Comment:
should trigger points to an existing component or create a new component on the flight?

Reply:
Just followed the example code on https://react.semantic-ui.com/modules/popup/
Not sure how to point to an existing component.

Reply:
yeah. it creates trigger component too.  it seems a wrapper so the trigger is a wrappered component.